src/data/languages/languageData.ts

```diff
@@ -0,0 +1,32 @@
+import { ProductKey } from '../types';
+import { LanguageData } from './types';
+
+export default {
+  platform: {} as LanguageData,
+  pubsub: {
+    javascript: 1.2,
```

Comment:
Oh right, this might be why I kept being confused seeing JS 1.2 in the snapshots... Is this right though? I was certain we're on 2.x for JS

Reply:
This data is inaccurate atm, most of it is bot-vomited. Will need some fine tuning in the QA period with Mark etc but the key thing is that the data structures provide the framework we need